robust_intelligence_ai_firewall/README.md

```diff
@@ -0,0 +1,76 @@
+# Agent Check: Robust Intelligence AI Firewall
+
+## Overview
+
+[Robust Intelligence’s AI Firewall][1] wraps a protective layer around your AI models. It helps:
+1. **Block malicious inputs in real time.** Attacks on AI systems are increasing in frequency and sophistication. The nature of large language models (LLMs) make them a disproportionally high target, but all model types are at risk. AI Firewall inspects every input and automatically blocks malicious payloads before they can do damage to your model. Risks includes prompt injection, prompt extraction, and PII exfiltration.
+2. **Validate model outputs in real time.** AI models will inevitably generate undesired responses due to both malicious and inadvertent user actions. AI Firewall scans model outputs to ensure they are free of sensitive information, hallucinations, or otherwise harmful content. Responses that fall outside your organization’s standards will be blocked from the application. This includes sensitive data from fine-tuning or connected databases used for retrieval-augmented generation.
+
+This integration monitors the AI Firewall results through the Datadog Agent. This includes metrics for allowed datapoints, blocked datapoints, and reasons data points were blocked.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Robust Intelligence AI Firewall check on your host. See [Use Community Integrations][2] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+   ```shell
+   datadog-agent integration install -t datadog-robust-intelligence-ai-firewall==<INTEGRATION_VERSION>
```

Comment:
please include the integration version in the command the customer will run

Reply:
Added it